app/main.py

```diff
@@ -1,8 +1,84 @@
+class CarValidationError(Exception):
+    """This Exception is for errors when initializing Car object"""
+
+
+class CarWashStationError(Exception):
+    """This Exception is for errors when initializing CarWashStation object"""
```

Comment:
In practice, if your exceptions are not self-explanatory, you don't need to pass a doc. But here I guess we need to, but write a doc from a 3d person perspective (there's no `This`)
There's error with initialization of  ...

Reply:
Thank you!